addons/remaining-replies/userscript.js

```diff
@@ -0,0 +1,54 @@
+export default async function ({ addon, msg }) {
+  function addRemainingReplyCount(comment) {
+    // skip the main comment input at the top of the page
+    if (comment.classList.contains("compose-row")) return;
+    // commenting was turned off for the studio
+    if (!comment.querySelector(".comment-reply span")) return;
+
+    let parentComment = comment.parentElement.classList.contains("replies")
+      ? comment.parentElement.parentElement.children[0]
+      : comment;
+
+    const parentCommentID = parentComment.getAttribute("id");
+    const parentCommentData = addon.tab.redux.state.comments.comments.filter(
+      // substring in order to remove the "comments-" prefix
+      (comment_in_list) => comment_in_list.id == parentCommentID.substring(9)
+    )[0];
+
+    const remainingReplies = 25 - parentCommentData?.reply_count;
+
+    const span = document.createElement("span");
```

Comment:
Shouldn't it check first if such span already exists in the comment?
This function "addRemainingReplyCount" can be called more than once on the same comment. It appears to be intentional.
This causes a bug